lib/ansible/modules/network/netconf/netconf_config.py

```diff
@@ -398,7 +431,7 @@ def main():
                 conn.lock(target=target)
                 locked = True
             if before is None:
-                before = to_text(conn.get_config(source=target), errors='surrogate_then_replace').strip()
+                before = to_text(conn.get_config(target, filter_spec), errors='surrogate_then_replace').strip()
```

Comment:
I have some concern about your change from keyword arg to positional args. While this may work it's a code smell. I'd rather see `get_config(source=target, filter=filter_spec)` if that will work for you.

Reply:
:nose:

Reply:
I knows what you did there.